configs/chicoma.json

```diff
@@ -1,21 +1,21 @@
 {
-    "cms_dir": "/users/moraru/Parsl-Project/exa-amd/cms_dir",
-    "work_dir": "/lustre/scratch5/moraru/work_dir_v1",
-    "cpu_account": "t25_ml-amd",
-    "gpu_account": "t25_ml-amd_g",
+    "cms_dir": "<path_to>/exa-amd/cms_dir",
+    "work_dir": "./work_dir_v1",
+    "cpu_account": "cpu_account",
+    "gpu_account": "gpu_account",
     "elements": "Na-B-C",
     "formation_energy_threshold": -0.2,
     "num_workers": 128,
-    "initial_structures_dir":"/users/moraru/Parsl-Project/exa-amd/cms_dir/mpstrs",
+    "initial_structures_dir":"initial_structures",
 
     "parsl_config": "chicoma_debug",
 
     "cgcnn_batch_size": 256,
 
-    "vasp_std_exe": "/usr/projects/icapt/applications/vasp/vasp-6.4.2-nvidia-gpu/bin/vasp_std",
-    "vasp_work_dir": "/users/moraru/Parsl-Project/vasp_work_dir_v2",
-    "vasp_pot_dir": "/users/moraru/Parsl-Project/data/select_structures/potpaw",
-    "vasp_output_file": "vasp_calc_result_1.csv",
+    "vasp_std_exe": "vasp_std",
```

Comment:
Do you need the full path to the VASP executable here? Where did you specify the path to the bin directory otherwise?